src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,316 @@
+  </figcaption>
+</figure>
+
+There's no option to bulk-select all contacts so that users are encouraged
+to select only the contacts that they need to share for that particular
+website. Users can also control which properties are shared with the site
+by toggling the property button at the top of the picker.
+
+<div class="clearfix"></div>
+
+### Transparency {: #security-transparency }
+
+To clarify which contact details are being shared, the picker always
+shows the contact's name and icon, plus any properties that the site has
+requested. For example, if a site requests `name`, `email`, and `tel`,
+all three properties will be shown in the picker. Alternatively,
+if a site only requests `tel`, the picker will show only the name, and
+telephone numbers.
+
+<figure class="w-figure w-figure--inline-left" style="max-width:42%">
+  <img  src="contact-picker-left.jpg">
+  <figcaption>
+    Picker, site requesting <code>name</code>, <code>email</code>, and
+    <code>tel</code>, one contact selected.
+  </figcaption>
+</figure>
+<figure class="w-figure w-figure--inline-right" style="max-width:42%">
+  <img  src="contact-picker-right.jpg">
+  <figcaption>
+    Picker, site requesting only <code>tel</code>, one contact selected.
+  </figcaption>
+</figure>
+
+<div class="clearfix"></div>
+
+<figure class="w-figure w-figure--inline-right" style="max-width:50%">
+  <img  src="contact-picker-long-press.jpg">
+  <figcaption>
+    The result of a long press on a contact.
+  </figcaption>
+</figure>
+
+A long press on a contact will show all of the information that will be
+shared if the contact is selected. (See the Cheshire Cat contact image.)
+
+<div class="clearfix"></div>
+
+### No permission persistence {: #security-persistence }
+
+Access to contacts is on-demand, and not persisted. Each time a site wants
+access, it must call `navigator.contacts.select()` with a user gesture,
+and the user must individually choose the contact(s) they want to share
+with the site.
+
+## Feedback {: #feedback }
+
+We want to hear about your experiences with the Contact Picker API.
+
+### Tell us about the API design
+
+Is there something about the API that doesn't work as expected? Or
+are there missing methods or properties that you need to implement your idea?
+
+* File a spec issue on the [WICG Contact Picker API GitHub repo][issues],
+  or add your thoughts to an existing issue.
+
+### Problem with the implementation?
+
+Did you find a bug with Chrome's implementation? Or is the implementation
+different from the spec?
+
+* File a bug at [https://new.crbug.com][new-bug]. Be sure to include as much
+  detail as you can, provide simple instructions for reproducing the bug, and
+  set *Components* to `Blink>Contacts`. [Glitch](https://glitch.com) works great
+  for sharing quick and easy repros.
+
+### Planning to use the API?
+
+Are you planning to use the Contact Picker API? Your public support helps us
```

Comment:
```suggestion
Are you planning to use the Contact Picker API? Your public support helps the Chrome team
```

Avoiding "we" per Rob's rationale in https://github.com/GoogleChrome/web.dev/pull/1614/files#r333143069